### PR TITLE
Accessibility enhancement

### PR DESCRIPTION
**Pull request by aizjicod**
on this pull request, we were checking the accessibility of the page. These were the results.

- title- no changes required.
- img - add alternative text.
- headline - change the disposition of the headlines.
- color contrast - no changes requiered.
- text-zooming - change height in several parts and added a flex-wrap on card-works.
- Interaction - no changes requiered.
- Moving content - no changes required.
- multimedia - no changes requiered.
- The basic structure of the page - no changes are required.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -41,10 +41,10 @@
...
       <article class="pageSection titleText">
         <h1 class="stText">
-          Hey there. I’m Ja'fer
-        </h1>
-        <h1 class="ndText">
-          I’m a software developer
+          Hey there. I’m Ja'fer <br>
+          <span class="ndText">
+            I’m a software developer
+          </span>
```
Comment (aizjicod) [inline index.html:47]: change h1 to span.

Comment (aizjicod) [inline index.html:47]: change h1 to span

File: index.html [hunk 2]
```diff
@@ -72,8 +72,8 @@ <h2 id="h1WorkTitle">My Recent Works</h2>
   <section class="gridContainer">
     <div class="gridCell works">
       <article class="cardContainer">
-        <img src="images/Img_Plaholder.svg" alt="" class="imgPlaceholder">
-        <p class="projectTitle">Multi-Post Stories Gain+Glory</p>
+        <img src="images/Img_Plaholder.svg" alt="Image placeholder" class="imgPlaceholder">
+        <h3 class="projectTitle">Multi-Post Stories Gain+Glory</h3>
```
Comment (aizjicod) [inline index.html:76]: change p tag to h3 tag

File: index.html [hunk 3]
```diff
@@ -85,8 +85,8 @@ <h2 id="h1WorkTitle">My Recent Works</h2>
     </div>
     <div class="gridCell work">
       <article class="cardContainer">
-        <img src="images/Img_Plaholder.svg" alt="" class="imgPlaceholder">
-        <h2 class="projectTitle">Multi-Post Stories Gain+Glory</h2>
+        <img src="images/Img_Plaholder.svg" alt="Image placeholder" class="imgPlaceholder">
+        <h3 class="projectTitle">Multi-Post Stories Gain+Glory</h3>
```
Comment (aizjicod) [inline index.html:89]: change h2 tag to an h3 tag.

File: index.html [hunk 4]
```diff
@@ -98,8 +98,8 @@ <h2 class="projectTitle">Multi-Post Stories Gain+Glory</h2>
...
```
Comment (aizjicod) [inline index.html:102]: change h2 tag to an h3 tag.

File: index.html [hunk 5]
```diff
@@ -111,8 +111,8 @@ <h2 class="projectTitle">Multi-Post Stories Gain+Glory</h2>
...
```
Comment (aizjicod) [inline index.html:115]: change h2 tag to an h3 tag.

File: index.html [hunk 6]
```diff
@@ -124,8 +124,8 @@ <h2 class="projectTitle">Multi-Post Stories Gain+Glory</h2>
...
```
Comment (aizjicod) [inline index.html:128]: change h2 tag to an h3 tag.

File: index.html [hunk 7]
```diff
@@ -137,8 +137,8 @@ <h2 class="projectTitle">Multi-Post Stories Gain+Glory</h2>
...
```
Comment (aizjicod) [inline index.html:141]: change h2 tag to an h3 tag.

File: index.html [hunk 8]
```diff
@@ -161,8 +161,8 @@ <h2 class="h2aboutMe">About me
   <div class="aboutMeCardCont">
     <section class="gridContainerAM">
       <div class="cardAM">
-        <img src="images/icon-languages.svg" alt="" class="imageAmCard">
-        <h2 class="cardAMTitle">Languages</h2>
+        <img src="images/icon-languages.svg" alt="Languages Icon" class="imageAmCard">
+        <h3 class="cardAMTitle">Languages</h3>
```
Comment (aizjicod) [inline index.html:165]: change h2 tag to an h3 tag.

File: index.html [hunk 9]
```diff
@@ -171,8 +171,8 @@ <h2 class="cardAMTitle">Languages</h2>
         </ul>
       </div>
       <div class="cardAM">
-        <img src="images/icon-frameworks.svg" alt="" class="imageAmCard">
-        <h2 class="cardAMTitle">Frameworks</h2>
+        <img src="images/icon-frameworks.svg" alt="Frameworks Icon" class="imageAmCard">
+        <h3 class="cardAMTitle">Frameworks</h3>
```
Comment (aizjicod) [inline index.html:175]: change h2 tag to an h3 tag.

File: index.html [hunk 10]
```diff
@@ -182,8 +182,8 @@ <h2 class="cardAMTitle">Frameworks</h2>
         </ul>
       </div>
       <div class="cardAM">
-        <img src="images/icon-skills.svg" alt="" class="imageAmCard">
-        <h2 class="cardAMTitle">Skills</h2>
+        <img src="images/icon-skills.svg" alt="Skills Icon" class="imageAmCard">
+        <h3 class="cardAMTitle">Skills</h3>
```
Comment (aizjicod) [inline index.html:186]: change h2 tag to an h3 tag.

**Review by aizjicod — COMMENTED** [review]
accessibility for text heading.

File: index.html [hunk 11]
```diff
@@ -54,14 +54,14 @@ <h1 class="ndText">
...
-        <li> <a href="#" class="smIcon"><img src="images/twitter.svg" alt=""></a> </li>
-        <li> <a href="#" class="smIcon"><img src="images/Linkedin_icon.svg" alt=""></a> </li>
-        <li> <a href="#" class="smIcon"><img src="images/medium.svg" alt=""></a> </li>
-        <li> <a href="#" class="smIcon"><img src="images/github.svg" alt=""></a> </li>
-        <li> <a href="#" class="smIcon"><img src="images/angellist.svg" alt=""></a> </li>
+        <li> <a href="#" class="smIcon"><img src="images/twitter.svg" alt="Twitter Link"></a> </li>
+        <li> <a href="#" class="smIcon"><img src="images/Linkedin_icon.svg" alt="LinkedIn Link"></a> </li>
+        <li> <a href="#" class="smIcon"><img src="images/medium.svg" alt="Medium Link"></a> </li>
+        <li> <a href="#" class="smIcon"><img src="images/github.svg" alt="GitHub Link"></a> </li>
+        <li> <a href="#" class="smIcon"><img src="images/angellist.svg" alt="Angell Link"></a> </li>
```
Comment (aizjicod) [inline index.html:61]: alternative text added.

File: index.html [hunk 12]
```diff
@@ -54,14 +54,14 @@ <h1 class="ndText">
...
       </ul>
       <div class="nextPageIcon">
-        <a href=""><img src="images/Disabled.svg" alt=""></a>
+        <a href=""><img src="images/Disabled.svg" alt="Next Page"></a>
```
Comment (aizjicod) [inline index.html:64]: alternative text added.

File: index.html [hunk 13]
```diff
@@ -72,8 +72,8 @@ <h2 id="h1WorkTitle">My Recent Works</h2>
   <section class="gridContainer">
     <div class="gridCell works">
       <article class="cardContainer">
-        <img src="images/Img_Plaholder.svg" alt="" class="imgPlaceholder">
-        <p class="projectTitle">Multi-Post Stories Gain+Glory</p>
+        <img src="images/Img_Plaholder.svg" alt="Image placeholder" class="imgPlaceholder">
```
Comment (aizjicod) [inline index.html:75]: alternative text added.

File: index.html [hunk 14]
```diff
@@ -85,8 +85,8 @@ <h2 id="h1WorkTitle">My Recent Works</h2>
     </div>
     <div class="gridCell work">
       <article class="cardContainer">
-        <img src="images/Img_Plaholder.svg" alt="" class="imgPlaceholder">
-        <h2 class="projectTitle">Multi-Post Stories Gain+Glory</h2>
+        <img src="images/Img_Plaholder.svg" alt="Image placeholder" class="imgPlaceholder">
```
Comment (aizjicod) [inline index.html:88]: alternative text added.

File: index.html [hunk 15]
```diff
@@ -98,8 +98,8 @@ <h2 class="projectTitle">Multi-Post Stories Gain+Glory</h2>
...
```
Comment (aizjicod) [inline index.html:101]: alternative text added.

File: index.html [hunk 16]
```diff
@@ -111,8 +111,8 @@ <h2 class="projectTitle">Multi-Post Stories Gain+Glory</h2>
...
```
Comment (aizjicod) [inline index.html:114]: alternative text added.

File: index.html [hunk 17]
```diff
@@ -124,8 +124,8 @@ <h2 class="projectTitle">Multi-Post Stories Gain+Glory</h2>
...
```
Comment (aizjicod) [inline index.html:127]: alternative text added.

File: index.html [hunk 18]
```diff
@@ -137,8 +137,8 @@ <h2 class="projectTitle">Multi-Post Stories Gain+Glory</h2>
...
```
Comment (aizjicod) [inline index.html:140]: alternative text added.

File: index.html [hunk 19]
```diff
@@ -161,8 +161,8 @@ <h2 class="h2aboutMe">About me
   <div class="aboutMeCardCont">
     <section class="gridContainerAM">
       <div class="cardAM">
-        <img src="images/icon-languages.svg" alt="" class="imageAmCard">
-        <h2 class="cardAMTitle">Languages</h2>
+        <img src="images/icon-languages.svg" alt="Languages Icon" class="imageAmCard">
```
Comment (aizjicod) [inline index.html:164]: alternative text added.

File: index.html [hunk 20]
```diff
@@ -171,8 +171,8 @@ <h2 class="cardAMTitle">Languages</h2>
         </ul>
       </div>
       <div class="cardAM">
-        <img src="images/icon-frameworks.svg" alt="" class="imageAmCard">
-        <h2 class="cardAMTitle">Frameworks</h2>
+        <img src="images/icon-frameworks.svg" alt="Frameworks Icon" class="imageAmCard">
```
Comment (aizjicod) [inline index.html:174]: alternative text added.

File: index.html [hunk 21]
```diff
@@ -182,8 +182,8 @@ <h2 class="cardAMTitle">Frameworks</h2>
         </ul>
       </div>
       <div class="cardAM">
-        <img src="images/icon-skills.svg" alt="" class="imageAmCard">
-        <h2 class="cardAMTitle">Skills</h2>
+        <img src="images/icon-skills.svg" alt="Skills Icon" class="imageAmCard">
```
Comment (aizjicod) [inline index.html:185]: alternative text added.

**Review by aizjicod — COMMENTED** [review]
improved image alternative text for accessibility.

File: styles.css [hunk 22]
```diff
@@ -158,6 +158,7 @@ a {
   list-style-type: none;
   width: 327px;
   flex-direction: row;
+  flex-wrap: wrap;
```
Comment (aizjicod) [inline styles.css:161]: add flex-wrap for manipulating the items inside of the flex-box.

File: styles.css [hunk 23]
```diff
@@ -275,7 +276,6 @@ a {
   border: 1px solid #c1c7d0;
   border-radius: 8px;
   width: 327px;
-  height: 352px;
```
Comment (aizjicod) [inline styles.css:278]: delete height for manipulating the text.

File: styles.css [hunk 24]
```diff
@@ -322,7 +322,6 @@ a {
 
 .cf_text {
   width: 327px;
-  height: 363px;
```
Comment (aizjicod) [inline styles.css:325]: delete height for manipulating the text.

File: styles.css [hunk 25]
```diff
@@ -426,7 +425,6 @@ button:hover {
 
   .page1Container {
     background-image: url("images/Header-llustration-desktop.svg");
-    height: 100vh;
```
Comment (aizjicod) [inline styles.css:429]: delete height for manipulating the text.

**Review by aizjicod — COMMENTED** [review]
manipulating the style of the page in order to upgrade the accessibility.

**Review by tresorsawasawa — APPROVED** [review]
# STATUS: APPROVE :heavy_check_mark: :heavy_check_mark:  :champagne: 

Hi Team
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**